app/borrowers/server.py

```diff
@@ -0,0 +1,27 @@
+from flask import jsonify
+
+
+def register_routes(blueprint):
+    @blueprint.route('/get-borrower/<borrower_id>', methods=['GET'])
```

Comment:
can we follow a REST approach, so the url describes the thing not the action. So it would be `/borrower/<id>` rather than `/get-borrower/<id>`.